ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-07.txt

```diff
@@ -49,144 +70,198 @@ Copyright Notice
    carefully, as they describe your rights and restrictions with respect
    to this document.  Code Components extracted from this document must
    include Simplified BSD License text as described in Section 4.e of
+   the Trust Legal Provisions and are provided without warranty as
+   described in the Simplified BSD License.
+
+Table of Contents
+
+   1.  Introduction  . . . . . . . . . . . . . . . . . . . . . . . .   2
+   2.  Motivation  . . . . . . . . . . . . . . . . . . . . . . . . .   3
+   3.  BDP metadata parameters . . . . . . . . . . . . . . . . . . .   4
+   4.  Extension activation  . . . . . . . . . . . . . . . . . . . .   5
+   5.  Discussion  . . . . . . . . . . . . . . . . . . . . . . . . .   5
+   6.  Acknowledgments . . . . . . . . . . . . . . . . . . . . . . .   6
+   7.  IANA Considerations . . . . . . . . . . . . . . . . . . . . .   6
+   8.  Security Considerations . . . . . . . . . . . . . . . . . . .   6
+   9.  Informative References  . . . . . . . . . . . . . . . . . . .   6
+   Appendix A.  Example of server solution . . . . . . . . . . . . .   7
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .   8
+
+1.  Introduction
+
+   Each transport connection typically starts without knowledge of the
+   path between the endpoints.  Transport protocols use implicit signals
+   from the network to discover the properties of the path.  This
+   information is used to adapt the transport mechanisms to the network
+   path.  For example, an Internet transport endpoint is unable to
+   determine a safe rate at which to start or continue their
+   transmission, and uses slow-start to determine a safe rate.  This
+   applies to the 1-RTT mode of QUIC.
 
+   QUIC supports the sending of data in two different modes, after the
+   transport handshake has completed, 1-RTT mode, and sending data along
+   with handshake packets, 0-RTT mode.  By using 0-RTT data an
+   application is able to send data with the handshake packets, making
```

Comment:
/send data/send transport parameters/

Reply:
Correct in commit ""send transport parameters"